tooling/minimize/tests/pass/union.rs

```diff
@@ -20,4 +60,25 @@ fn main() {
     }
 
     let _y = B { f2: 0 };
+
+    // We don't care if the read value is correct, only whether reading from it is UB.
+    // Calling the function twice forces a copy.
```

Comment:
Move vs Copy makes no difference here, I think I'd prefer to remove that -- it just makes the test more confusing.

Reply:
Okay. I am not fully aware of the details of move vs copy semantics for copy objects, so I wanted to make sure that the union padding gets removed due to a literal copy,